src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1309,17 +1325,20 @@ private bool AddOrganelle(OrganelleTemplate organelle)
     private bool MoveOrganelle(OrganelleTemplate organelle, Hex oldLocation, Hex newLocation, int oldRotation,
         int newRotation)
     {
+        // TODO: consider allowing rotation inplace
+
         // Make sure placement is valid
         if (!IsMoveTargetValid(newLocation, newRotation, organelle))
             return false;
 
         // If the organelle was already moved this session, added (placed) this session,
         // or not moved (but can be rotated), then moving it is free
         bool isFreeToMove = organelle.MovedThisSession || oldLocation == newLocation || organelle.PlacedThisSession;
-        int cost = isFreeToMove ? 0 : Constants.ORGANELLE_MOVE_COST;
+        int moveCost = (int)(Constants.ORGANELLE_MOVE_COST * editorCostFactor);
+        int cost = isFreeToMove ? 0 : moveCost;
 
         // Too low mutation points, cancel move
-        if (!isFreeToMove && Editor.MutationPoints < Constants.ORGANELLE_MOVE_COST)
+        if (!isFreeToMove && Editor.MutationPoints < moveCost)
```

Comment:
I know you are just updating existing code here, but wouldn't it make more sense to use the `cost` variable on this line? So this if would be: `if (Editor.MutationPoints < cost)` as cost is set to 0 anyway if isFreeToMove is true.